### ISSUE TITLE
jobs/css/bootstrap.min.css values conflict with css/main.css

### ISSUE THREAD

**Issue by evalica** [opensourcedesign/opensourcedesign.github.io#90]
The job form uses a full bootstrap.min.css that has different defaults than main.css. 

Some example: 
- .navbar-nav>li>a: padding from 20px to 15px
- .navbar-nav>li>a: font-size from 16px to 14px
- etc.

This is visible here for example: 
- Job Form nav: 
<img width="500" alt="jobform" src="https://cloud.githubusercontent.com/assets/629552/24585053/cc16fa8e-1789-11e7-95f5-ee0fcbbb65ef.png">

- Main nav:
<img width="500" alt="main" src="https://cloud.githubusercontent.com/assets/629552/24585057/d957dd26-1789-11e7-87ee-0ef541b75e8d.png">

I'm sure other values also conflict. 

Solutions: 
1. see if jobs/css/bootstrap.min.css can be removed or what it brings in addition. those values can be added in main.css
2. use the same Bootstrap values in jobs/css/bootstrap.min.css as main.css

**Comment by gaja6413** [comment]
Thankyou @evalica  :)
Even I have noticed this issue.
Using the same values will bring the consistent to the website.
Is there any other view on this issue? or will I change the values

**Comment by evalica** [comment]
It should be investigated why that file was added in the first place.

**Comment by simonv3** [comment]
I'm fairly sure it uses bootstrap to do most of its styling. If that file were removed we'd lose a lot of the basic styles - which is not necessarily a bad thing, it's a lot of styles for a simple web form.

**Comment by bnvk** [comment]
Yep. This should be harmonized to be using the same global CSS styles and template in mail site repo!

**Comment by Erioldoesdesign** [comment]
Is this still an issue? can it be closed?

**Comment by AnXh3L0** [comment]
Not applicable anymore since new site is a mono-repo, and it's using Hugo + Tailwind, closing.
